routes/util.js

```diff
@@ -1,14 +1,29 @@
+const { getUserById } = require("../db")
+
 function isLoggedIn(req, res, next) {
     if(!req.user) {
         res.status(401);
         next({
             name: "MissingUserError",
             message: "You must be logged in"
         });
-    }
+    };
+    next();
+};
+
+function isAdmin(req, res, next) {
+    const _user = getUserById(req.id)
```

Comment:
almost! instead of passing the id in `req.id` from the client, since you include the user id and username when you [sign the token here](https://github.com/hoguer/porto3000/blob/25ff8dfb876b12b40cc73676597e7dd52b4aaacb/routes/users.js#L43), you can add a middleware function so that you have them available on every request. 
Here you go: https://github.com/hoguer/porto3000/pull/59/commits/b9c51e3be09dc9078e7b1a0cb4bac1980d16de86